test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -225,7 +225,12 @@ public async Task DotnetRestore_ProjectMovedDoesNotRunRestore()
                     var xml = XDocument.Load(stream);
 
                     var attributes = new Dictionary<string, string>() { { "Version", "1.0.0" } };
-
+#if IS_NETCORE30
```

Comment:
Is this a specific test?

Reply:
This is to fix a broken test in netcoreapp3.0 only (netcoreapp2.2 won't fail). As there is no more “NuGetFallbackFolder” in SDK3.0,  running "dotnet restore" (with SDK version 3.0)on project with TFM property of netstandar2.0 would fail. So adding this to change the TFM from netstandard2.0 to netstandard2.1 in project file when testing with netcoreapp3.0.

Reply:
makes sense.